pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
actually, even worse -- `options` doesn't appear to ever be set to non-`nil` within this function